.travis.yml

```diff
@@ -0,0 +1,29 @@
+language: python
+python: 3.6
+
+env:
+  - DOCKER_COMPOSE_VERSION=1.26.0
+
+before_install:
+  - sudo rm /usr/local/bin/docker-compose
```

Comment:
The default version of docker compose is the one with the random strings appended to containers, hence installing a newer version here.